importers/djornl/parser.py

```diff
@@ -239,22 +300,24 @@ def load_edges(self):
         node_name = self.config('node_name')
         # these functions remap the values in the columns of the input file to
         # appropriate values to go into Arango
+        # note that the functions that assume the presence of a certain key in the input
+        # can do so because that key is in a 'required' property in the CSV spec file
         remap_functions = {
             # create a unique key for each record
-            '_key': lambda row: '__'.join([row[_] for _ in ['node1', 'node2', 'layer_descrip', 'edge']]),
+            '_key': lambda row: '__'.join([row[_] for _ in ['node1', 'node2', 'edge_type', 'score']]),
             'node1': None,  # this will be deleted in the 'store' step
             'node2': None,  # as will this
             '_from': lambda row: node_name + '/' + row['node1'],
             '_to': lambda row: node_name + '/' + row['node2'],
-            'score': lambda row: float(row['edge']),
-            'edge_type': lambda row: row['layer_descrip'],
+            'score': lambda row: float(row['score']),
+            'edge_type': None,
         }
 
         # store edge data, checking for potential duplicates
         def store_edges(datum):
             # there should only be one value for each node<->node edge of a given type,
             # so use these values as an index key
-            edge_key = "__".join([datum['node1'], datum['node2'], datum['edge_type']])
+            edge_key = "__".join([*sorted([datum['node1'], datum['node2']]), datum['edge_type']])
```

Comment:
Ensures that edge A-->B of type Z is recognised as being the same as edge B-->A of type Z as the edge direction is meaningless in this context.